### PR TITLE
Task #34695 import CSV file

### PR DESCRIPTION
**Pull request by phuongbk**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -8,4 +8,11 @@ class User < ActiveRecord::Base
   def is_guest?
     id.nil?
   end
+
+  def self.import(file)
```
Comment (biznezo) [inline app/models/user.rb]: try to remove ()

**Comment by yuyuvn** [comment]
LGTM

File: app/models/user.rb [hunk 2]
```diff
@@ -8,4 +8,11 @@ class User < ActiveRecord::Base
...
+    CSV.foreach(file.path, headers: true) do |row|
+      User.create row.to_hash.merge password: 
+        Devise.friendly_token.first(Settings.user.password_length)
```
Comment (biznezo) [inline app/models/user.rb]: ()

**Comment by yuyuvn** [comment]
LGTM

**Pull request merged**